src/routes/paths.ts

```diff
@@ -21,7 +21,11 @@ export const getGivePath = () => '/give';
 export const getMapPath = () => '/map';
 export const getVouchingPath = () => '/vouching';
 export const getHistoryPath = () => '/history';
-export const getAdminPath = () => '/admin';
+export const getAdminPath = () => '/circles';
+export const getAdminPath1 = () => '/admin';
```

Comment:
Maybe we can just hide the path to this new admin interface

Reply:
if you want to automatically hide it in production and show it in all other environments, you can check the `IN_PRODUCTION` env var in `src/utils/domain.ts`